test/FunctionalTests/Microsoft.OData.Client.Tests/Microsoft.OData.Client.Tests.csproj

```diff
@@ -37,6 +37,10 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.6.0" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp1.1' or '$(TargetFramework)' == 'net452'">

```

Comment:
why do we still support 'netcoreapp1.1'?

Reply:
Seems that this PR is targeted at 7.x branch.